projects/dsp-ui/src/lib/viewer/values/date-value/date-value.component.spec.ts

```diff
@@ -611,5 +633,24 @@ describe('DateValueComponent', () => {
 
   });
 
+  describe('create a date value no required value', () => {
+
+    let testHostComponent: TestHostCreateValueNoValueRequiredComponent;
+    let testHostFixture: ComponentFixture<TestHostCreateValueNoValueRequiredComponent>;
+
+    beforeEach(() => {
+      testHostFixture = TestBed.createComponent(TestHostCreateValueNoValueRequiredComponent);
+      testHostComponent = testHostFixture.componentInstance;
+      testHostFixture.detectChanges();
+
+      expect(testHostComponent).toBeTruthy();
+      expect(testHostComponent.inputValueComponent).toBeTruthy();
+    });
+
+    it('should not create an empty value', () => {
```

Comment:
Maybe also check that the form is valid.

Reply:
done in c05e0fd